app/customer.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+
+@dataclass()
```

Comment:
This way to use `dataclass` decorator is OK but the following way is more common:
```suggestion
@dataclass
```